.github/workflows/e2e-main.yml

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: e2e-tests-main
+
+on:
+  workflow_run:
+    workflows: ["CI"]
```

Comment:
This is a trigger to this workflow based on the status of the job named "CI". This workflow does not exist here. This section should be removed and instead, ie. `schedule` section can be added to run workflow on schedule or if there is a workflow that runs after each merge it can be used instead of CI.

Reply:
I think there is not a workflow that runs after each merge (the closest one I found was `release.yaml` but I think it's not the same) so I've scheduled it to run at midnight

Reply:
This one is: https://github.com/crc-org/crc-extension/blob/main/.github/workflows/build-builder.yaml#L20. In theory, if you run the e2e tests just after some change was merge AND built and image published, your `latest` tag used in e2e tests would be testing actual published change within the extension.

Reply:
But it can be follow up issue. I am ok with that.